.ci/ansible/elastic-stack.yml

```diff
@@ -49,3 +49,16 @@
       - { old: 'http://fleet-server', new: 'http://{{ inventory_hostname }}', path: "{{ e2e_base_dir }}cli/config/compose/profiles/fleet/default/kibana.config.yml" }
       - { old: 'http://kibana', new: 'http://{{ inventory_hostname }}', path: "{{ e2e_base_dir }}cli/config/compose/services/elastic-agent/fleet-server/docker-compose.yml" }
       - { old: 'http://package-registry:8080', new: 'https://epr-staging.elastic.co', path: "{{ e2e_base_dir }}cli/config/compose/profiles/fleet/default/kibana.config.yml" }
+
+  - name: Start stack without any tag to spin up the stack using Fleet's bootstrapping code
+    become: True
+    ansible.builtin.shell:
+    args:
+      cmd: "TAGS='non-existing-tag' {{ e2e_base_dir }}.ci/scripts/functional-test.sh"
+
+  - name: Wait for the stack to come up
+    wait_for: host={{ inventory_hostname }} port={{ item.port }} delay=10 timeout=60
```

Comment:
120 to match the description?

Reply:
The changes in the PR are not related to that. It's weird to explain offline, as we have found that fleet-server starts in the tests too fast, and we do not know what to wait for on Kibana before starting the fleet-server.